jetty9-base/pom.xml

```diff
@@ -112,7 +136,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/jetty-base/lib/gcp</outputDirectory>
+              <outputDirectory>${project.build.directory}/jetty-base/lib/gcloud</outputDirectory>
```

Comment:
Ah yes, I remember now .... as the jetty-base jar is to do with running jetty on gcp and nothing to do with gcloud datastore sessions, we decided to split the libs.